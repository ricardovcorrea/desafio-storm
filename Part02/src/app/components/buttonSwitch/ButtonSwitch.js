import React from "react";
import "./ButtonSwitch.scss";

const ButtonSwitch = () => {
  return <div></div>;
};

export default ButtonSwitch;
