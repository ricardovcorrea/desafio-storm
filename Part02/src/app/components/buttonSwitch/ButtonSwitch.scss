.buttonSwitchGroup {
  margin: 0px;
  height: 50px;
}

.buttonSwitch {
  border: 0px !important;
  background-color: #fff !important;
  box-shadow: 0px 1px 5px 3px rgba(0,0,0,0.08);
  color: #676767 !important;
}

.buttonSwitchSelected {
  box-shadow: none !important;
  background: rgb(216, 51, 103);
  background: linear-gradient(
    45deg,
    rgba(216, 51, 103, 1) 13%,
    rgba(253, 131, 2, 1) 85%
  );
  margin-left: -2px !important;
}

.buttonSwitchSelectedIcon {
  color: white;
}
