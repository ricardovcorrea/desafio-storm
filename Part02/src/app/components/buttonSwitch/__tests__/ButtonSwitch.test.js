import React from 'react';
import { render } from '@testing-library/react';
import ButtonSwitch from '../ButtonSwitch';

test('renders button switch', () => {
  const container = render(<ButtonSwitch />)
  expect(container.container).toMatchSnapshot()
});
