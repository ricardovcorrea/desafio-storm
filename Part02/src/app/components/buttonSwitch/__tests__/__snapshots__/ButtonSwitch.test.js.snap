// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button switch 1`] = `
<div>
  <div />
</div>
`;
